### PR TITLE
feat: E2E testing

### LINKED ISSUE #709: E2E testing
### Please describe the problem that your request should resolve.
Implement solution to early detect regressions of the most common flows

### Describe the solution you'd like
- [x] research of solution
- [x] identify user flows to cover
- [x] implementation
- [x] infrastructure settings

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #709 

## Demo
The infrastructure is not properly set yet.
The desired state is to run E2E test after push to PR.
After merging this, I will info Petar so he can debug it from develop

However the test can be run locally with 
`cypress run`
or 
`cypress open` for GUI intereaction

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add e2e tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,9 +116,10 @@ Supports [evergreen](https://nuxt.com/blog/v3#the-browser-and-nodejs-support) br
 The core browser set targets the 2 most recent major versions of Chrome, Firefox, and Edge on a monthly basis and Safari
 on a yearly basis.
 
-## Tooling
+## Testing
 
-This project uses [Applitools Visual AI](https://applitools.com/) as part of its automated testing.
+This project uses [Applitools Visual AI](https://cypress.io/) for e2e testing.
+Locally can be run by ```cypress open```
```
Comment (michele-franchi) [inline README.md:122]:
```suggestion
This project uses [Cypress](https://cypress.io/) for e2e testing.
Locally can be run using the command ```yarn e2e:open```.
```

File: README.md [hunk 2]
```diff
@@ -134,7 +135,7 @@ get involved!
 Stay up-to-date on the latest developments in the æternity ecosystem by following us on social media.
 
 - [Forum](https://forum.aeternity.com/)
-- [X](https://x.com/aeternity)
+  - [X](https://x.com/aeternity)
```
Comment (michele-franchi) [inline README.md:138]:
```suggestion
- [X](https://x.com/aeternity)
```

**Review by michele-franchi — COMMENTED** [review]
I tested it locally and works great but I would suggest to add the following commands in the package.json to use the cypress package from node_modules:

```
"e2e:run": "cypress run",
"e2e:open": "cypress open"
```

**Comment by janmichek** [comment]
> I tested it locally and works great but I would suggest to add the following commands in the package.json to use the cypress package from node_modules:
> 
> ```
> "e2e:run": "cypress run",
> "e2e:open": "cypress open"
> ```

👍 added

**Comment by michele-franchi** [comment]
@janmichek I still don't see the suggested package.json commands in the changes. Could you check?

**Comment by janmichek** [comment]
> @janmichek I still don't see the suggested package.json commands in the changes. Could you check?

oops, wrong branch. Now it's fixed

**Review by michele-franchi — APPROVED** [review]
LGTM

**Pull request merged**